guides/writing-addons/addon-dependencies.md

```diff
@@ -0,0 +1,45 @@
+In an Ember project, the command `ember install <package-name>` installs addons and other npm packages as devDependencies in the top level `package.json`.
+
+Some project packages such `ember-cli-app-version` and `broccoli-asset-rev` are only used during development or the application build process. The package `ember-cli-htmlbars` is a dependency that must be included in a production build. 
+
+During the build process, the Ember CLI will only incorporate the addons and packages needed to distribute the application.
+
+With addons, the `dummy app` is the project and the addon will seem like a dependency of the dummy app. This makes it easy to use the dummy app for testing and documenting your addon's usage.
```

Comment:
This seems a bit out of place. Maybe it belongs lower down?

Reply:
moved this based on other changes